pkgs/by-name/qr/qrtool/package.nix

```diff
@@ -6,37 +6,42 @@
 , installShellFiles
 }:
 
-rustPlatform.buildRustPackage rec {
-  pname = "qrtool";
-  version = "0.8.5";
+let
+  name = "qrtool";
+  version = "0.10.0";
+in
+
+rustPlatform.buildRustPackage {
+  pname = name;
+  inherit version;
 
   src = fetchFromGitHub {
     owner = "sorairolake";
-    repo = "qrtool";
+    repo = name;
     rev = "v${version}";
-    sha256 = "sha256-jrvNZGO1VIDo6Mz3NKda1C7qZUtF9T00CAFK8yoGWjc=";
+    hash = "sha256-PXjggeDofrrS32XdqnOgAMlAiw+/7eMQpELw8EmiwNI=";
   };
 
-  cargoSha256 = "sha256-JOnvlabCr3fZsIIRc2qTjf50Ga83zL8Aoo2sqzMBs7g=";
+  cargoHash = "sha256-0Vwx80XxrJX8C293+4qgRr0pTMV6Jy3IBNQISzFxyr0=";
 
   nativeBuildInputs = [ asciidoctor installShellFiles ];
 
   postInstall = ''
     # Built by ./build.rs using `asciidoctor`
-    installManPage ./target/*/release/build/qrtool*/out/*.?
+    installManPage ./target/*/release/build/${name}*/out/*.?
```

Comment:
using `pname` in here and other places is an anti pattern.

eg if repo gets renamed tomorrow ( eg fork ) unlikely that all other references will also be renamed